urwid/_raw_display_base.py

```diff
@@ -262,18 +267,16 @@ def get_input(self, raw_keys: bool = False) -> list[str] | tuple[list[str], list
 
         # Avoid pegging CPU at 100% when slowly resizing
         if keys == ["window resize"] and self.prev_input_resize:
```

Comment:
technically this method is not used due to `hook_event_loop` API availability, but looks like kept for compatibility